docs/linux/zbd-api.md

```diff
@@ -248,26 +233,19 @@ enum blk_zone_cond {
 };
 ```
 
-Under a device normal operation, some of the conditions defined cannot result
-directly from a host initiated operation. These conditions are
-`BLK_ZONE_COND_OFFLINE` and `BLK_ZONE_COND_READONLY` and can only be set by the
-device itself to indicate zones with capabilities limited by a hardware defect.
+Under a device's normal operation, some of these conditions cannot result directly from host-initiated operations. These conditions are `BLK_ZONE_COND_OFFLINE` and `BLK_ZONE_COND_READONLY`. They can be set only by the device itself, to indicate zones with capabilities that have been limited by a hardware defect.
 
-The condition `BLK_ZONE_COND_EXP_OPEN`, or *explicit open*, is the result of a
-successful execution of an `OPEN ZONE` command
+The condition `BLK_ZONE_COND_EXP_OPEN` (or *explicit open*), is the result of the successful execution of an `OPEN ZONE` command
 (see [Zone Block Commands](../introduction/smr.md#zone-block-commands).
 
-Since the `OPEN ZONE` command is not supported by the kernel ZBD interface, a
-zone can be transitioned in the *explicit open* zone condition only by using
-direct device access, that is, issuing the SCSI `OPEN ZONE` command through the
-*SG_IO* interface (using *libzbc*, *libzbc zbc_open_zone* utility or the
-*sg_zone* utility).
+Because the `OPEN ZONE` command is not supported by the kernel ZBD interface, a
```

Comment:
Aouch ! That is not true since kernel 5.5... So we need a separate update for this. I will send something on top of this patch once it is merged.

Reply:
I'll leave this comment unresolved so that we can remember to fix this.